### ISSUE TITLE
OpenID module crashes server: ClassNotFoundException

### ISSUE THREAD

**Issue by pschichtel** [playframework/playframework#7629]
### Play Version (2.5.x / etc)
2.6.0 and 2.6.1 checked

### API (Scala / Java / Neither / Both)
Scala (2.12.2)

### Operating System (Ubuntu 15.10 / MacOS 10.10 / Windows 10)
```
Linux phillip-desktop 4.11.9-1-ARCH #1 SMP PREEMPT Wed Jul 5 18:23:08 CEST 2017 x86_64 GNU/Linux
```

### JDK (Oracle 1.8.0_72, OpenJDK 1.8.x, Azul Zing)

```
openjdk version "1.8.0_131"
OpenJDK Runtime Environment (build 1.8.0_131-b11)
OpenJDK 64-Bit Server VM (build 25.131-b11, mixed mode)
```

### Library Dependencies
OpenID module is relevant here.

### Expected Behavior

1. OpenID support works

### Actual Behavior

1. Call to ```OAuth.retrieveRequestToken(String)``` crashes the server

Class ```play.shaded.oauth.oauth.signpost.OAuth``` is trying to access ```com.google.gdata.util.common.base.PercentEscaper``` which does not exist, resulting in ```java.lang.ClassNotFoundException: com.google.gdata.util.common.base.PercentEscaper```

### Reproducible Test Case

```Class.forName("play.shaded.oauth.oauth.signpost.OAuth")```

**Comment by pschichtel** [comment]
This is either a problem with relocations or with tree shaking, if that's applied

**Comment by wsargent** [comment]
This is coming from https://github.com/playframework/play-ws/blob/master/build.sbt#L210 because gdata is included in the signpost module directly under its own package.  I mistakenly thought it must be the packager acting up.

**Comment by richdougherty** [comment]
Thanks @wsargent. So since we need those `com.google.gdata` classes we shouldn't be `zap`ping them in the `play-ws` build. Zapping them removes them from the final artifact, which is the reason there's a missing class error. Is that right?

**Comment by pschichtel** [comment]
Still happening with play 2.6.2

Is something blocking this?

**Comment by marcospereira** [comment]
I'm reopening this since it will be fixed only after updating play-ws here.

**Comment by Uxio0** [comment]
Hi,

For those just looking for a simple fix, add this to your dependencies until it gets fixed:

`"com.google.gdata" % "core" % "1.47.1"`

Cheers!

**Comment by marcospereira** [comment]
Closed after #7668.
